shlr/gdb/meson.build

```diff
@@ -0,0 +1,34 @@
+# handle gdb dependency
+gdb_files = [
+  'src/arch.c',
+  'src/common.c',
+  'src/packet.c',
+  'src/utils.c',
+  'src/libgdbr.c',
+
+  'src/gdbclient/core.c',
+  'src/gdbclient/responses.c',
+  'src/gdbclient/xml.c',
+
+  'src/gdbserver/core.c',
+]
+
+gdb_inc = [
+  platform_inc,
+  include_directories(
+    './include',
+    './include/gdbclient',
+    './include/gdbserver'
+  )
+]
+
+libr2gdb = static_library('r2gdb', gdb_files,
+  include_directories: gdb_inc,
+  dependencies: [r_util_dep, r_cons_dep],
```

Comment:
drop cons

Reply:
Removing cons from there result in the following error in travis builds when compiling with `meson --prefix=/home/travis/build/radareorg/radare2/install -Db_coverage=true build`

```
shlr/gdb/libr2gdb.a(src_gdbclient_core.c.o): In function `gdbr_lock_enter':
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:94: undefined reference to `r_cons_break_push'
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:95: undefined reference to `r_cons_sleep_begin'
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:97: undefined reference to `r_cons_sleep_end'
shlr/gdb/libr2gdb.a(src_gdbclient_core.c.o): In function `gdbr_lock_leave':
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:105: undefined reference to `r_cons_break_pop'
shlr/gdb/libr2gdb.a(src_gdbclient_core.c.o): In function `gdbr_connect':
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:190: undefined reference to `r_cons_sleep_begin'
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:210: undefined reference to `r_cons_sleep_end'
shlr/gdb/libr2gdb.a(src_gdbclient_core.c.o): In function `send_vcont':
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:1274: undefined reference to `r_cons_sleep_begin'
/home/build/radareorg/radare2/build/../shlr/gdb/src/gdbclient/core.c:1289: undefined reference to `r_cons_sleep_end'
collect2: error: ld returned 1 exit status
```